commands/issue.go

```diff
@@ -710,3 +721,106 @@ func milestoneValueToNumber(value string, client *github.Client, project *github
 
 	return 0, fmt.Errorf("error: no milestone found with name '%s'", value)
 }
+
+func transferIssue(cmd *Command, args *Args) {
+	if args.ParamsSize() < 2 {
+		utils.Check(cmd.UsageError(""))
+	}
+
+	issueNumber := args.GetParam(0)
+	targetRepo := args.GetParam(1)
+
+	var issueID, repositoryID string
+
+	localRepo, err := github.LocalRepo()
+	utils.Check(err)
+
+	project, err := localRepo.MainProject()
+	utils.Check(err)
+
+	owner := project.Owner
+	currRepo := project.Name
+	host := project.Host
+	gh := github.NewClient(host)
+
+	query := fmt.Sprintf(`
+        query($issue: Int = %s, $owner: String = "%s", $repo: String = "%s") {
```

Comment:
Even though this works, the query should ideally not contain any values:
```suggestion
        query($issue: Int!, $owner: String!, $repo: String!) {
```
instead, pass values as `variables` parameter in JSON:
```go
variables := map[string]interface{}{
  "issue": issueNumber,
  "owner": owner,
  "repo": currRepo,
}
body["variables"] = variables
```